doc/rbd/iscsi-target-cli.rst

```diff
@@ -17,9 +17,9 @@ install, and configure the Ceph iSCSI gateway for basic operation.
 
    -  ``targetcli-2.1.fb47`` or newer package
 
-   -  ``python-rtslib-2.1.fb64`` or newer package
+   -  ``python-rtslib-2.1.fb67`` or newer package
```

Comment:
It's actually ```2.1.fb68``` [1]

[1] https://github.com/ceph/ceph-iscsi/pull/108

Reply:
Fixed